hack/docker_build.sh

```diff
@@ -14,11 +14,14 @@ tag_for_branch() {
 # push to kubespheredev with default latest tag
 REPO=${REPO:-kubespheredev}
 TAG=$(tag_for_branch $1)
+RELEASE=${2:-false}
```

Comment:
I don't see why `release` is necessary here, what you really need to change is `tag_for_branch` function, maybe return a default tag when no branch specified

Reply:
make build only build image 
make release pull image to repo

use release params to distinguish them